.travis.yml

```diff
@@ -27,8 +27,6 @@ script:
 matrix:
     fast_finish: true
     include:
-      - php: 7.2
-        env: SYMFONY_VERSION='^3.4.0'
```

Comment:
This bundle still supports Symfony 3.4, why was this removed?

Reply:
I guess because I misread it. If it can be re-added and the builds/tests pass, then great